recipes/ocpsvg/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ocpsvg" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/ocpsvg-{{ version }}.tar.gz
+  sha256: 8a089249b52b0bff99cca9698af3d74ef678ced2359316f8c2a04aee9baafb46
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.9
+    - cadquery
+    - ocp >=7.7.0
+    - svgpathtools >=1.5.1,<2
+    - svgelements >=1.9.1,<2
+
+test:
+  source_files:
+    - tests
+  imports:
+    - ocpsvg
+  commands:
+    - pytest -v tests/
+  requires:
+    - pytest
```

Comment:
```suggestion
    - pytest
    # The libgl dependency for the test section can likely be removed when
    # the following https://github.com/conda-forge/ocp-feedstock/pull/50
    # gets merged in the ocp feedstock
    # this helps avoid a yum dependency
    # either with the yum dependency or with this, the end user is assumed
    # to have X11 system dependencies installed
    # this helps the user avoid to have a direct dependency on libgl
    # from this package (it will be transitive from OCP once the PR gets merged)
    - libgl    # [linux]
```

Reply:
Is there anything written down about the "new GL dependency management"; I was under the impression it was a CDT because it _couldn't_ be packaged by conda-forge?

Reply:
you can read about the efforts in 
https://github.com/conda-forge/staged-recipes/pull/25919
and
https://github.com/conda-forge/staged-recipes/pull/27107

Reply:
but nothing more, still kinda in the testing for m.e

Reply:
Thank you, that is exactly the sort of thing I was looking for